### PR TITLE
support protocol 428

### PR DESCRIPTION
**Pull request by brokiem**
Tested and worked

### CODE REVIEW

**Comment by thebigsmileXD** [comment]
Is this complete now?

File: src/specter/Specter.php [hunk 1]
```diff
@@ -60,7 +60,6 @@ public function onCommand(CommandSender $sender, Command $command, string $label
                     } else {
                         return false;
                     }
-                    break;
```
Comment (thebigsmileXD) [inline src/specter/Specter.php:63]: hmm. not too sure about this one. i once heard that it is only safe to remove them if the "case" got parenthesis around it

Comment (dktapps) [reply src/specter/Specter.php:63]: It's unreachable because all branches in this block have return statements in them.

**Comment by brokiem** [comment]
I think this is complete

**Comment by inxomnyaa** [comment]
Thank you for the PR

**Pull request merged**